app/src/main/java/com/x8bit/bitwarden/data/vault/repository/util/DomainsExtensions.kt

```diff
@@ -6,14 +6,14 @@ import com.x8bit.bitwarden.data.vault.repository.model.DomainsData
 /**
  * Map the API [Domains] model to the internal [DomainsData] model.
  */
-fun Domains.toDomainsData(): DomainsData {
+fun Domains?.toDomainsData(): DomainsData {
```

Comment:
Even though we can have a null `Domains`, we always map it back to an empty _nonnull_ `DomainsData`.